mtgsqlive/json2sql.py

```diff
@@ -347,13 +398,16 @@ def handle_legal_rows(
     """
     legalities = []
     for card_format, format_status in card_data["legalities"].items():
-        legalities.append(
-            {"uuid": card_uuid, "format": card_format, "status": format_status}
-        )
+        # I use an assumed "legal" system to save some space/queries, only insert rows with non-legal status
+        # If a query returns no results you can assume its legal in the format queried.
+        # "Innocent Until Proven Guilty" :D
```

Comment:
_ehhhhhh_ not sure if I agree with this

Reply:
That's why I commented this to explain it, it's really a matter of opinion. I'll leave it up to you.

Reply:
Yeah, I'm gonna want all entries. Some data can be inferred, this isn't easy to infer.